### ISSUE TITLE
Room List should act a navigation treeview widget

### ISSUE THREAD

**Issue by t3chguy** [vector-im/riot-web#11205]
As per the ARIA recommendations we have implemented the Room List as a Navigation Tree View, the only thing we don't conform to is the entire tree should have just one tabindex, for the active element, the navigation should be done by only the arrow keys.

This would be jarring to anyone used to existing behaviour.

> Room list is even more awesome now and the fact new room and new chat buttons are reachable from the keyboard is another positive change. Thank you!
> Regarding the room list interaction it would perhaps be possible to tweak it a little bit more in a way so tabIndex="0" would be removed from the treeview items and only up and down arrow keys would be navigating over the items. Pressing tab would jump past the tree and pressing shift+tab would move before the tree. Would this be doable please?

from @pvagner

**Comment by MarcoZehe** [comment]
FWIW I agree with @pvagner on this.

**Comment by t3chguy** [comment]
Most of the work here is done but the entire widget should have just one tab stop, the selected room tile and use only arrow-keys for navigation.

https://www.w3.org/TR/wai-aria-practices/examples/treeview/treeview-2/treeview-2a.html
